gamestonk_terminal/portfolio/port_controller.py

```diff
@@ -45,59 +49,41 @@ def __init__(self):
         self.broker_list = set()
         self.merged_holdings = None
 
-    @staticmethod
-    def print_help(broker_list):
-
+    @classmethod
+    def print_help(cls, broker_list):
         """Print help"""
-        print("\nBrokers Supported: rh   - Robinhood")
-        print("                   alp  - Alpaca")
-        print("                   ally - Ally Invest")
-        print("                   dg - Degiro")
-        print("\nPortfolio:")
-        print("   help          show this menu again")
-        print(
-            "   q             quit this menu, and shows back to main menu, logs out of brokers"
-        )
-        print("   quit          quit to abandon program, logs out of brokers")
 
         print(
-            f"\nCurrent Broker: {('None', ', '.join(broker_list))[bool(broker_list)]}"
+            "Brokers Supported:\n"
```

Comment:
I think this brokers supported can be removed, as we specify them below

Reply:
How would they know the commands for login (`rh`, `alp`, `ally`) ?

Reply:
We could have on the main menu:

Robinhood (rh)
Alpaca (alp)

and so on

Reply:
Leaving it like this : as discussed on `Discord`.